src/qt/rpcconsole.cpp

```diff
@@ -468,6 +468,7 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
 
     // Install event filter for up and down arrow
     ui->lineEdit->installEventFilter(this);
+    ui->lineEdit->setMaxLength(INT_MAX);
```

Comment:
Suggest `std::numeric_limits<int32_t>::max()` as it's the standard for size limits.  I'm also not really sure a 2GB buffer makes sense here, but then again, what is a sane amount? Somewhere between 2^20 and 2^31 seems fine, but towards the latter seems a bit overboard.

Reply:
thanks @tryphe changed for the standard size limit. Yes, hard question about what number should be there. 
What rpc command should be the largest one? Maybe importblock with a full block?
16MB should be fine?

Reply:
I think it's hard to answer that. If we make it just big enough for the biggest command we can come up with, someone will come up with a bigger one. I also don't see any way someone could accidentally paste 2GB of data in the textbox, which at worst would just make things a bit slow. I opt to wait for more comments to see if anyone objects.